cime_config/machines/cmake_macros/gnugpu_crusher.cmake

```diff
@@ -0,0 +1,27 @@
+string(APPEND FFLAGS " -Wno-implicit-interface")
+
+if (NOT DEBUG)
+  string(APPEND CFLAGS   " -O2")
+  string(APPEND CXXFLAGS " -O2")
+  string(APPEND FFLAGS   " -O2")
+endif()
+if (COMP_NAME STREQUAL gptl)
+  string(APPEND CPPDEFS " -DHAVE_SLASHPROC")
+endif()
+set(NETCDF_PATH "$ENV{NETCDF_DIR}")
+set(PNETCDF_PATH "$ENV{PNETCDF_DIR}")
+set(PIO_FILESYSTEM_HINTS "gpfs")
+string(APPEND CMAKE_OPTS " -DPIO_ENABLE_TOOLS:BOOL=OFF")
+
+set(MPICC  "cc")
+set(MPICXX "hipcc")
+set(MPIFC  "ftn")
+set(SCC  "cc")
+set(SCXX "hipcc")
+set(SFC  "ftn")
+
+string(APPEND CXXFLAGS " -I$ENV{MPICH_DIR}/include --amdgpu-target=gfx90a")
```

Comment:
I'm probably missing something trivial. This flag is causing issues with PIO build for me as PIO is trying to use this flag with Cray CC. Have you seen anything similar? As nightly tests worked, I wonder what in my environment is causing PIO to pickup CC?